lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -317,13 +317,14 @@
 .forum-nav-thread-wrapper-1 {
     @extend %forum-nav-thread-wrapper;
     margin: 0 ($baseline / 4);
-    width: 80%;
+    max-width: calc(100% - 125px);
```

Comment:
The 125 is how may pixels I determined the indicator that there are unread responses could take up (assuming we never go higher than 99 unread responses). There might be a better way to do this in flexbox, though.